packages/design-system/src/components/Button/Button.stories.jsx

```diff
@@ -72,3 +73,13 @@ IconButton.args = {
     </>
   ),
 };
+
+export const SpinnerButton = Template.bind({});
+SpinnerButton.args = {
+  children: (
+    <>
+      <Spinner /> Loading...
```

Comment:
I had initially imported the `DefaultSpinner` story for this, but since it appears the Button overwrites Spinner props (can't change sizing or color), I imported the Spinner component itself.

Reply:
will it work without the fragment wrapper? It's adds the <Fragment> to the code example.. no big if not.

Reply:
Unfortunately not, because of the sibling "Loading..." text you'd get a warning that there needs to be a wrapping container. I experimented with [changing `docs.source.type` with a parameter](https://storybook.js.org/docs/react/writing-docs/doc-blocks#docspage-1), but it seems like the fragment's going to be stuck there.